### DISCUSSION TITLE
How to change header height when using createNativeStackNavigator

### DISCUSSION THREAD

**Discussion by hariks789** [software-mansion/react-native-screens#832] [Q&A] [answered]
Header height is really small in iPads. Is there a way to set custom height.
![Screenshot 2021-02-24 at 7 51 26 PM](https://user-images.githubusercontent.com/22391855/109013933-af8a7680-76d9-11eb-889f-61f6b18eb191.png)

**Comment by WoLewicki** [comment, marked as answer]
`Native-stack` uses default platform header height and is not customizable at this moment. You can provide `headerLargeTitle` if you want the bigger title style on the screen.
